client/src/pages/ApplicationsPage/ApplicationsPage.tsx

```diff
@@ -22,14 +39,17 @@ const ApplicationsPage = (): JSX.Element => {
     }
 
     fetchApplications();
-  }, []);
+    // TODO Not sure I want to keep this. here for testing
+    const intervalId = setInterval(fetchApplications, 5000);
```

Comment:
I don't think we want this - refetching is already triggered by changes to `showRejected` and `dateFilter`

Reply:
I tend to agree. It is there so I can see updates take effect in real-time once records pass the time threshold. Once we move to production with intervals measured in days it doesn't make much sense.

Reply:
we gotta hunt down that page refresh bug!